src/tile-graphics.ts

```diff
@@ -0,0 +1,377 @@
+import { Graphics, Point } from 'pixi.js'
+import { TileModel, RoadType, RoadAxis } from './tile-model'
+
+enum RoadSide {
+  Top,
+  Right,
+  Bottom,
+  Left
+}
+
+class TileGraphics {
+  tileWidth: number
+  roadWidth: number
+
+  tileCornerRadius: number = 20
+  roadArcRadius: number = 20
+
+  roadLineWidth: number = 4
+  dashLineWidth: number = 2
+
+  trainDashSpacing: number = 16
+  trainDashLength: number = 18
+  carDashLength: number = 12
+
+  constructor(tileWidth: number, roadWidth: number) {
+    this.tileWidth = tileWidth
+    this.roadWidth = roadWidth
+  }
+
+  make = (tile: TileModel, existing?: Graphics): Graphics => {
+    const graphics = existing || new Graphics()
+    graphics.clear()
+
+    graphics.lineStyle(this.roadLineWidth, 0x000000, 1.0)
+    graphics.beginFill(0xffffff)
+    graphics.drawRoundedRect(0, 0, this.tileWidth, this.tileWidth, this.tileCornerRadius)
+    graphics.endFill()
+
+    this.drawRoadSegments(tile, graphics)
+    this.drawCenterCrossIfNecessary(tile, graphics)
+    this.drawCenterBlockIfNecessary(tile, graphics)
+
+    return graphics
+  }
+
+  // MARK: - Helpers
+
+  private drawRoadSegments = (tile: TileModel, graphics: Graphics) => {
+    const drawCarRoad = this.curriedDrawCarRoad(tile, graphics)
+    const drawTrainRoad = this.curriedDrawTrainRoad(tile, graphics)
+
+    if (tile.top == RoadType.Car) {
+      drawCarRoad(RoadSide.Top)
+    } else if (tile.top == RoadType.Train) {
+      drawTrainRoad(RoadSide.Top)
```

Comment:
Train Road!

Reply:
lol. Definitely open to renaming if it's silly. Maybe something like `path`?

Reply:
Nah. I think this makes total sense